src/ts/waterfall/details-overlay/html-details-body.ts

```diff
@@ -129,7 +138,7 @@ export function createDetailsBody(requestID: number, entry: WaterfallEntry, acco
       </div>
       ${timingsTab}
       <div class="tab raw-data">
-        <pre><code>${JSON.stringify(entry.rawResource, null, 2)}</code></pre>
+        <pre><code>${escapeHtml(JSON.stringify(entry.rawResource, null, 2))}</code></pre>
```

Comment:
I hope I understand the question correct. Yes, it could contain quotes, but as far as I can remember they are just rendered as html content, not attributes, so it's just a concern from a XSS perspective (plan to look into this later) - or did I overlook anything?